### ISSUE TITLE
Substitute crashes when V/Oct input polyphony is >8

### ISSUE THREAD

**Issue by anlexmatos** [squinkylabs/SquinkyVCV#191]
Running Rack v1.1.6 on Windows 7.

This crash leaves no trace in Rack's crash log, but I was able to get this message out of the debug console in VS Code:

```
Thread 33 received signal SIGSEGV, Segmentation fault.
0x0000000015699214 in plugin!_ZN15SimpleQuantizer8quantizeEf () from C:\Users\admin\DOCUME~1\Rack\plugins-v1\squinkylabs-plug1\plugin.dll
```

**Comment by squinkylabs** [comment]
Interesting. thank you

**Comment by anlexmatos** [comment]
Here's the full call stack I'm seeing in VS Code:

![image](https://user-images.githubusercontent.com/12039110/106375936-3a37ba00-6356-11eb-9273-d7086024d2a9.png)

**Comment by squinkylabs** [comment]
oh, interesting, thank you. I assume you can work around this by limiting the polyphony? Substitute polyphony only works up to 8 anyway.

**Comment by squinkylabs** [comment]
Fixed in 1.0.17, which is now in the library.
